src/component/axis/CartesianAxisView.js

```diff
@@ -137,7 +142,7 @@ var CartesianAxisView = AxisView.extend({
                     y2: p2[1]
                 },
                 style: zrUtil.defaults({
-                    stroke: lineColors[colorIndex]
+                    stroke: (viewLabels && viewLabels[i] && !viewLabels[i].formattedLabel ? 'transparent' : lineColors[colorIndex])
```

Comment:
For better performance, I think we should not draw transparent line, if condition were matched, just don't execute this iteration.

Reply:
I was worried about causing side effects before, so I did n’t do that.

now the iteration not execute with ```continue```

Reply:
I don't think there gonna have side affect,  `silent` might concerning but it has been set to `true`, which won't dispatch event.